gcp_variant_transforms/libs/vcf_header_parser.py

```diff
@@ -42,7 +42,7 @@ def get_vcf_headers(input_file):
   try:
     vcf_reader = vcf.Reader(fsock=_line_generator(input_file))
   except (SyntaxError, StopIteration) as e:
-    raise ValueError('Invalid VCF header: %s' % str(e))
+      raise ValueError('Invalid VCF header: %s in: %s' % (str(e), input_file))
```

Comment:
```suggestion
      raise ValueError('Invalid VCF header in %s: %s' % (input_file, str(e)))
```

nit: this is to have a consistent error similar to [here](https://github.com/googlegenomics/gcp-variant-transforms/blob/master/gcp_variant_transforms/beam_io/vcf_parser.py#L359)